.github/pull_request_template.md

```diff
@@ -1,10 +1,21 @@
+Please, update this template according to your PR requirements.
+
 Fixes #
 
 Changes proposed in this pull-request:
--
--
--
+- Upgraded ...
+- Pipeline validation ...
+- Documented ...
+- Example ...
+- Script ...
```

Comment:
I think you are missing other descriptions such as bug-fix, or feature. Would be a good idea to provide an example of a PR in every case. Also, you can point out to the different labels that are available, perhaps we should review the labels as well.

Reply:
I think that is contemplated at the upper line `Fixes #`

For me the `-` are points that the one writing the PR description should fill in. So this template is a guide/reminder of which points should be written. Not meaning that all of them should.

Making it too long could be counter-productive.

About the labels, perhaps this could be defined at the contribution guide, then we review the labels on GitHub.